.github/workflows/test.yml

```
@@ -47,7 +47,7 @@ jobs:
     - name: Install dependencies
       run: |
         poetry run pip install --upgrade pip setuptools
-        poetry install
+        poetry install --no-root
```

We need the project as well here
```suggestion
        poetry install
```